galoy.yaml

```diff
@@ -102,6 +108,10 @@ rateLimits:
     points: 8
     duration: 3600
     blockDuration: 86400
+  createDeviceAccountPerIp:
```

Comment:
this is currently not used right now. probably dropped in some refactoring. this was implemented pre appcheck. maybe this could be dropped for now?

Reply:
I guess we're using `checkFailedLoginAttemptPerIpLimits` instead which is good.